ocs_ci/ocs/cluster.py

```diff
@@ -1362,6 +1362,61 @@ def check_ceph_osd_tree_after_node_replacement():
     return True
 
 
+def silence_ceph_osd_crash_warning(osd_pod_name):
+    """
+    Silence the osd crash warning of a specific osd pod
+
+    Args:
+        osd_pod_name (str): The name of the osd pod which we need to
+            silence the crash warning
+
+    Returns:
+        bool: True if it found the osd crash with name 'osd_pod_name'. False otherwise
+
+    """
+    ct_pod = pod.get_ceph_tools_pod()
+    new_crash_objects_list = ct_pod.exec_ceph_cmd(ceph_cmd="ceph crash ls-new")
+    for crash_obj in new_crash_objects_list:
+        if crash_obj.get("utsname_hostname") == osd_pod_name:
+            logger.info(f"Found osd crash with name {osd_pod_name}")
+            obj_crash_id = crash_obj.get("crash_id")
+            logger.info("silence the osd crash warning")
```

Comment:
Before silence, the osd crash warning can you please add a logger.info for `ceph crash info <crash_id>` to track info if needed

Reply:
Done